### PR TITLE
Unscripted tests for SANS GUI v2

### PR DESCRIPTION
**Pull request by keeeto**
**Description of work.**

**Report to:** nobody

**To test:**

* Build a dev docs target of Mantid `dev-docs-html`
* In `dev-docs/html/Testing` open `index.html`
* Check that there is a link to `SANS GUI Testing`
* Follow the link and check that the following page makes sense

*There is no associated issue.*

*This does not require release notes* because it is not a user-facing change.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: dev-docs/source/Testing/SANSGUI/SANSGUITests.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+-  Get the ISIS sample data from the website
+-  Ensure that the ISIS sample data directory is in the search directories
+
+
+**Time required 20 - 30 minutes**
+
+--------------
+
+#. Open ``Interfaces`` > ``SANS`` > ``SANS v2``
+#. Set ``Instrument`` to ``LOQ``
```
Comment (TTitcombe) [inline dev-docs/source/Testing/SANSGUI/SANSGUITests.rst]: When PR #24128 merges, Instrument will no longer be setable, instead it is determined by the User File.

Comment (keeeto) [reply dev-docs/source/Testing/SANSGUI/SANSGUITests.rst]: Good spot Tom. Is that PR approved and ready to go? In which case let's just hold fire on this one until it's done.

File: dev-docs/source/Testing/SANSGUI/SANSGUITests.rst [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+	- Click ``Process All``
+	- This should result in a plot with three lines
+#. In the ``Beam centre`` tab
+	- Click Run
+	- A plot should appear after some seconds, with 4 lines
+	- The 4 lines should gradually get closer together
+	- This will run for some time, probably minutes
+#. In the ``Sum Runs`` tab
+		- Enter ``74044, 74019`` in the top line
+		- Enter ``LOQ74044-add`` as Output file
```
Comment (TTitcombe) [inline dev-docs/source/Testing/SANSGUI/SANSGUITests.rst]: Need to click `add` before clicking `sum`. Once you click add the Output file name will be populated

**Review by TTitcombe — CHANGES_REQUESTED** [review]
Small change for sum runs step, otherwise a good script that seems clear to follow

**Review by TTitcombe — APPROVED** [review]

**Comment by TTitcombe** [comment]
This PR provides a thorough, easy to follow script for testing/using SANS

**Pull request merged**